src/main/java/com/voltskiya/misc/players/crafting/RegisterCrafting.java

```diff
@@ -0,0 +1,80 @@
+package com.voltskiya.misc.players.crafting;
+
+public class RegisterCrafting {
+
+    public static void registerCraftingRecipes() {
+        recipeFilter();
+        recipeChain();
+        recipeWool();
+        recipeChainArmor();
+    }
+
+    private void recipeChain(){
+        ShapedRecipe recipe = new ShapedRecipe(new NamespacedKey(VoltskiyaPlugin.get(), "Chain"), new ItemStack(Material.CHAIN, 2));
+        recipe.shape("N", "I", "N");
+        recipe.setIngredient('I', Material.IRON_INGOT);
+        recipe.setIngredient('N', Material.IRON_NUGGET);
+        Bukkit.addRecipe(recipe);
+    }
+    private void recipeWool(){
+        ShapedRecipe recipe = new ShapedRecipe(new NamespacedKey(VoltskiyaPlugin.get(), "Wool"), new ItemStack(Material.STRING, 4));
+        recipe.shape("W");
+        recipe.setIngredient('W', new RecipeChoice.MaterialChoice(woolList()));
+        Bukkit.addRecipe(recipe);
+    }
+    private void recipeChainArmor(){
+        ShapedRecipe helmet = new ShapedRecipe(new NamespacedKey(VoltskiyaPlugin.get(), "ChainHelmet"), new ItemStack(Material.CHAINMAIL_HELMET, 1));
+        helmet.shape("CCC", "C C");
+        helmet.setIngredient('C', Material.CHAIN);
+        Bukkit.addRecipe(helmet);
+
+        ShapedRecipe chestPlate = new ShapedRecipe(new NamespacedKey(VoltskiyaPlugin.get(), "ChainChestPlate"), new ItemStack(Material.CHAINMAIL_CHESTPLATE, 1));
+        chestPlate.shape("C C", "CCC", "CCC");
+        chestPlate.setIngredient('C', Material.CHAIN);
+        Bukkit.addRecipe(chestPlate);
+
+        ShapedRecipe leggings = new ShapedRecipe(new NamespacedKey(VoltskiyaPlugin.get(), "ChainLeggings"), new ItemStack(Material.CHAINMAIL_LEGGINGS, 1));
+        leggings.shape("CCC", "C C", "C C");
+        leggings.setIngredient('C', Material.CHAIN);
+        Bukkit.addRecipe(leggings);
+
+        ShapedRecipe boots = new ShapedRecipe(new NamespacedKey(VoltskiyaPlugin.get(), "ChainBoots"), new ItemStack(Material.CHAINMAIL_BOOTS, 1));
+        boots.shape("C C", "C C");
+        boots.setIngredient('C', Material.CHAIN);
+        Bukkit.addRecipe(boots);
+    }
+
+    private void recipeFilter(){
+        Iterator<Recipe> recipes = Bukkit.recipeIterator();
+        List<Material> bannedItems = new ArrayList<>();
+        bannedItems.add(Material.CHAIN);
+        bannedItems.add(Material.BUCKET);
+        while (recipes.hasNext()) {
+            if (bannedItems.contains(recipes.next().getResult().getType())) {
+                recipes.remove();
+            }
+        }
+    }
+
+    private List<Material> woolList() {
```

Comment:
Since we're in Java 17, we can do List.of() to make lists as well. 
```return List.of(Material.WOOL, Material.WOOL, Material.WOOL, Material.WOOL, Material.WOOL);```
This is perfectly fine as is though